packages/react-core/src/components/TextArea/examples/TextAreaValidated.tsx

```diff
@@ -31,21 +30,17 @@ export const TextAreaValidated: React.FunctionComponent = () => {
   };
   return (
     <Form>
-      <FormGroup
-        label="Comments:"
-        type="string"
-        helperText={helperText}
-        helperTextInvalid={invalidText}
-        fieldId="selection"
-        validated={validated}
-      >
+      <FormGroup label="Comments:" type="string" fieldId="selection">
         <TextArea
           value={value}
-          onChange={value => handleTextAreaChange(value)}
+          onChange={(value) => handleTextAreaChange(value)}
           isRequired
           validated={validated}
           aria-label="invalid text area example"
         />
+        <HelperText>
+          <HelperTextItem variant={validated}>{helperText}</HelperTextItem>
+        </HelperText>
```

Comment:
This is an issue on the live site as well so not blocking for this PR, but may want to open a followup issue regarding the below visual bug:

https://user-images.githubusercontent.com/70952936/225994814-3da945f7-5d95-42bb-a567-1263af321ce4.mov

Video shows the error text "You're being too brief, please enter at least 10 characters" being displayed briefly before updating to "Thanks for your comments!" despite no additional characters being added.

Reply:
I'm guessing this may have to do with the "simulated network call" timeout that's included here, but I'm unsure. Opened #8847 to track that.